lib/travis/build.rb

```diff
@@ -10,6 +10,11 @@ module Build
     BUILD_DIR = File.join(HOME_DIR, 'build')
 
     class << self
+      def version
+        @version ||= `git rev-parse HEAD 2>/dev/null || \\
+                        echo "${SOURCE_VERSION:-fafafaf}"`.strip
```

Comment:
`fafafaf` (and `fffffff`), to me, mean "unknown according to us", as opposed to being unknown according to something we don't control like a 3rd party lib or Heroku itself.  Yes, it's a bit special.  We are the primary consumers of this, so I consider the use of `fafafaf` to be a combination of useful and silly that is not harmful.  Please allow the `fafafaf`.